examples/provider/provider-sm.tf

```
@@ -25,17 +25,14 @@ resource "grafana_cloud_api_key" "metrics_publish" {
 resource "grafana_synthetic_monitoring_installation" "sm_stack" {
   provider = grafana.cloud
 
-  stack_id              = grafana_cloud_stack.sm_stack.id
-  metrics_instance_id   = grafana_cloud_stack.sm_stack.prometheus_user_id
-  logs_instance_id      = grafana_cloud_stack.sm_stack.logs_user_id
-  metrics_publisher_key = grafana_cloud_api_key.metrics_publish.key
+  stack_id = grafana_cloud_stack.sm_stack.id
 }
 
 // Step 3: Interact with Synthetic Monitoring
 provider "grafana" {
   alias           = "sm"
   sm_access_token = grafana_synthetic_monitoring_installation.sm_stack.sm_access_token
-  sm_url          = "<synthetic-monitoring-api-url>"
+  sm_url          = "grafana_synthetic_monitoring_installation.sm_stack.stack_sm_api_url"
```

Shouldn't this be an attribute reference instead?
```suggestion
  sm_url          = grafana_synthetic_monitoring_installation.sm_stack.stack_sm_api_url
```

Same for the files in `docs/`.